ci/github-script/merge.js

```diff
@@ -37,13 +38,20 @@ function runChecklist({
       .map(({ user }) => user.id),
   )
 
+  const trusted_eligible = trusted_maintainers.intersection(eligible)
+
   const checklist = {
     'PR targets a [development branch](https://github.com/NixOS/nixpkgs/blob/-/ci/README.md#branch-classification).':
       classify(pull_request.base.ref).type.includes('development'),
     'PR touches only packages in `pkgs/by-name/`.': allByName,
     'PR is at least one of:': {
       'Approved by a committer.': committers.intersection(approvals).size > 0,
+      'Approved by a trusted maintainer.':
```

Comment:
Similar to "development branches", I feel like "trusted maintainer" would benefit from linking to either the team's page or some docs describing what a "trusted maintainer" actually is.

Especially as this a new concept that almost everyone will be unfamiliar with.